src/azure-cli/azure/cli/command_modules/acs/_validators.py

```diff
@@ -614,3 +617,16 @@ def validate_registry_name(cmd, namespace):
         if pos == -1:
             logger.warning("The login server endpoint suffix '%s' is automatically appended.", acr_suffix)
             namespace.acr = registry + acr_suffix
+
+
+def validate_os_sku(namespace):
+    os_sku = namespace.os_sku
+    if os_sku in [CONST_OS_SKU_MARINER, CONST_OS_SKU_CBLMARINER]:
+        logger.warning(
+            'The options "%s" and "%s" will be renamed to %s, but "%s" and "%s" are still available.',
+            CONST_OS_SKU_MARINER,
+            CONST_OS_SKU_CBLMARINER,
+            CONST_OS_SKU_AZURELINUX,
+            CONST_OS_SKU_MARINER,
+            CONST_OS_SKU_CBLMARINER,
```

Comment:
```suggestion
            'The osSKU "%s" should be used going forward instead of "%s" or "%s". The osSKUs "%s" and "%s" will eventually be deprecated.',
            CONST_OS_SKU_AZURELINUX,
            CONST_OS_SKU_CBLMARINER,
            CONST_OS_SKU_MARINER,
            CONST_OS_SKU_CBLMARINER,
            CONST_OS_SKU_MARINER,
```

Reply:
Sure, updated.